docs/Documentation/Systems/Kestrel/index.md

```diff
@@ -1,5 +1,42 @@
 ---
+layout: default
 title: Kestrel
+parent: Systems
+has_children: true
+hide:
+ - toc
 ---
 
-Kestrel documentation coming soon. 
+# About the Kestrel Cluster
+
+Kestrel is configured to run compute-intensive and parallel computing jobs. It is a cluster comprised of 2454 nodes (servers) that run the Linux operating system (Red Had Enterprise Linux), with a peak performance of 44 PetaFLOPS.
+
+Please see the [System Configurations](../index.md) page for more information about hardware, storage, and networking.
+
+!!! note
+    GPUs are not currently available on Kestrel.  132 nodes with 4x Nvidia H100 GPUs are expected to be installed on Kestrel in FY24 Q2 (January, 2024).   
+## Accessing Kestrel
+Access to Kestrel requires an NREL HPC account and permission to join an existing allocation. Please see the [System Access](https://www.nrel.gov/hpc/system-access.html) page for more information on accounts and allocations.
+
+#### For NREL Employees:
+
+Users on an NREL device may connect via ssh to **kestrel.hpc.nrel.gov** from the NREL network. This will connect to one of the three login nodes. Users also have the option of connecting directly to an individual login node using one of the following names: 
+
+* kl1.hpc.nrel.gov
+* kl2.hpc.nrel.gov
+* kl3.hpc.nrel.gov
+
+#### For External Collaborators:
+If you are an external HPC user, you will need a [One-Time Password Multifactor token (OTP)](https://www.nrel.gov/hpc/multifactor-tokens.html) for two-factor authentication.
+
+!!! note
+    The external login nodes are not active currently.  For access to Kestrel at this time, you can connect the [HPC VPN](https://www.nrel.gov/hpc/vpn-connection.html), and then connect to one of the following names:
+
+    * kl1.hpc.nrel.gov
```

Comment:
Hostnames are the same as internal users once connected to the VPN or ssh gateway, so could just put connect to one of the hostnames as above instead of writing them out again.